config/services_test.yaml

```diff
@@ -0,0 +1,8 @@
+services:
+    task.reader.yaml:
+        class: SprykerSdk\Sdk\Infrastructure\Reader\TestTaskYamlReader
```

Comment:
move it to test folder. maybe helper if you need to replace some service

Reply:
Fixed